tests/cross/src/test/scala/tests/pc/PcDefinitionSuite.scala

```diff
@@ -211,7 +257,15 @@ class PcDefinitionSuite extends BasePcDefinitionSuite {
        |object Main {
        |  assert(/*scala/Predef.assert(). Predef.scala*/@@assertion = true)
        |}
-       |""".stripMargin
+       |""".stripMargin,
+    compat = Map(
+      "3.0" ->
+        """|
+           |object Main {
+           |  assert(/*scala/Predef.assert(+1). Predef.scala*/@@assertion = true)
```

Comment:
Is it a bug in Scala 2 implementation? It points to the first assert instead of the second

Reply:
I thought that is bug in Scala3 but everything is correct because there are stdLibPatches in scala3 https://github.com/lampepfl/dotty/issues/12942#issuecomment-868668468

The actual location should be [`scala/runtime/stdLibPatches/Predef.assert(+1)`](https://github.com/lampepfl/dotty/blob/master/library/src/scala/runtime/stdLibPatches/Predef.scala#L9) but it think it would be better to fix in separate PR. Probably we might overwrite such symbols somewhere in `DefitionProvider`

Reply:
Let's follow up with an issue then not to forget and add it to the comment here.